Lib/uuid.py

```diff
@@ -182,6 +196,7 @@ def __init__(self, hex=None, bytes=None, bytes_le=None, fields=None,
             int &= ~(0xf000 << 64)
             int |= version << 76
         self.__dict__['int'] = int
+        self.__dict__['is_safe'] = is_safe
```

Comment:
Do you want to assert the type of `is_safe` ? There seem to extensive checking of the rest of the parameters.
Not that I imagine people creating UUID object in their own code, but `is_safe` feel also a lot like a Boolean.